test/adjoint.jl

```diff
@@ -761,6 +830,13 @@ _, res = adjoint_sensitivities(sol_mm, alg, t = ts, dgdu_discrete = dg,
 reference_sol = ForwardDiff.gradient(p -> G(p, prob_mm, ts, sum), vec(p))
 @test res'≈reference_sol rtol=1e-11
 
+# TEST FAILS HERE. WORKS FOR LOWER TOLERANCE
+_, res_gauss = adjoint_sensitivities(sol_mm, alg, t = ts, dgdu_discrete = dg,
+    abstol = 1e-14, reltol = 1e-14,
+    sensealg = GaussAdjoint())
+@test_broken res_gauss≈res rtol=1e-11
+@test res_gauss≈res rtol=1e-3
```

Comment:
do you understand already why it doesn't work here? Saving/callback issues?

Reply:
I think it is a problem with the callback. I have noticed that it is not actually being called at every step of the integration. I am not entirely sure what is happening but I am looking into it now

Reply:
This was a separate issue with the IntegratingCallback, causing it to store double the amount of memory necessary. I have fixed this. I am still working on this issue